tests/unit/plugins/listops/append_operation.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2018 Weile Wei
+// Copyright (c) 2018 Hartmut Kaiser
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <phylanx/phylanx.hpp>
+
+#include <hpx/hpx_main.hpp>
+#include <hpx/util/lightweight_test.hpp>
+
+#include <cstdint>
+#include <string>
+
+///////////////////////////////////////////////////////////////////////////////
+phylanx::execution_tree::compiler::function compile(std::string const& code)
+{
+    phylanx::execution_tree::compiler::function_list snippets;
+    phylanx::execution_tree::compiler::environment env =
+        phylanx::execution_tree::compiler::default_environment();
+
+    return phylanx::execution_tree::compile(code, snippets, env);
+}
```

Comment:
Please verify this code against current master as the compiler API has changed slightly. You will have to adapt this test accordingly.